Readme.md

```diff
@@ -201,23 +202,60 @@ abbreviations should generally be avoided.
 
 ```js
 var adminUser = db.query('SELECT * FROM users ...');
+var itIsOkToHaveSuperLongVariableNames = 'whats up';
 ```
 
 *Wrong:*
 
 ```js
 var admin_user = db.query('SELECT * FROM users ...');
+var nua = 'Never use abbreviations.';
 ```
 
-## Use UpperCamelCase for class names
+#### Proper, well-known names should only capitalize the first character
+Proper or well-known abbreviations such as URL, XML, VAST, OpenRTB shoud only
+capitalize the first character of the abbreviation.
 
-Class names should be capitalized using `UpperCamelCase`.
+*Right:*
+
+```js
+var urlFormatter = 'is cool';
+var isUrlFormatter = 'is cool';
+var openRtbValue = 'is cool';
+var aUrlThatEatsXmlAndProducesOpenRtb = 'is cool';
+```
+
+*Wrong:*
+
+```js
+var uRLFormatter = 'is not cool';
+var URLFormatter = 'is not cool';
+var isURLFormatter = 'is not cool';
+var openRTBValue = 'is not cool';
+var aURLThatEatsXMLAndProducesOpenRTB = 'is not cool';
+```
+
+### Use UpperCamelCase for enum and class names
+
+Class names should be capitalized using `UpperCamelCase`. For example, when a
```

Comment:
:D "Class" in a sense when you append prototypical methods on a `function` object. I think it can be commonly understood what a class is representing in JavaScript.